### PR TITLE
feat(telephony.contact): update synchronization message for directory

### PR DESCRIPTION
**Pull request by Steffy29**
Ref: UXCT-170

### CODE REVIEW

File: client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html [hunk 1]
```diff
@@ -709,8 +709,8 @@
                         <label class="control-label"
                                data-translate="telephony_service_contact_status">
                         </label>
-                        <p class="form-control-static"
-                           data-ng-bind="'telephony_service_contact_sync' + (ServiceContactCtrl.directoryForm.status === 'todo' ? '_pending' : '') | translate">
+                        <p class="form-control-static">
```
Comment (antleblanc) [inline client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html]: Why not prefer using the `ngBind` directive?

```suggestion
                        <p class="form-control-static" data-ng-bind=":: ServiceContactCtrl.getStatusLabel()">
```

File: client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html [hunk 2]
```diff
@@ -709,8 +709,8 @@
                         <label class="control-label"
                                data-translate="telephony_service_contact_status">
                         </label>
-                        <p class="form-control-static"
-                           data-ng-bind="'telephony_service_contact_sync' + (ServiceContactCtrl.directoryForm.status === 'todo' ? '_pending' : '') | translate">
+                        <p class="form-control-static" data-ng-bind=":: ServiceContactCtrl.getStatusLabel()">
+                            {{ServiceContactCtrl.getStatusLabel()}}
```
Comment (marie-j) [inline client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html]: You don't need mustache templating with `ng-bind`

Comment (FredericEspiau) [reply client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html]: What Mary means is that mustache templating is worse for performances than directives:

```suggestion
                            <span data-ng-bind="ServiceContactCtrl.getStatusLabel()}"></span>
```

Comment (antleblanc) [reply client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html]: Please pay attention that the ngBind directive is already applied to the `<p>` element.

Could you please remove the extra `span` element?

Thanks!

File: client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.constant.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+angular.module('managerApp').constant('DIRECTORY_INFO', {
```
[the remark this answers is not in the document]
Comment (Steffy29) [reply client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.constant.js]: Yeah it make sense.

File: client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html (hunk 2 above)
Comment (marie-j) [reply client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html]: > What Mary means is that mustache templating is worse for performances than directives:

@FredericEspiau I meant that having both is not necessary. By the way it's Marie 😉

Comment (FredericEspiau) [reply client/app/telecom/telephony/service/contact/telecom-telephony-service-contact.html]: > > What Mary means is that mustache templating is worse for performances than directives:
> 
> @FredericEspiau I meant that having both is not necessary. By the way it's Marie 😉

What Frédérik means is that he's sorry he got your name wrong

**Pull request merged**